drv/sidecar-front-io/src/transceivers.rs

```diff
@@ -738,13 +1041,18 @@ impl Transceivers {
                 let has_err = (err & 0b1000) != 0;
                 if has_err {
                     match fpga_index {
-                        FpgaController::Left => out.left.set(port),
-                        FpgaController::Right => out.right.set(port),
+                        FpgaController::Left => physical_failure.left.set(port),
+                        FpgaController::Right => {
+                            physical_failure.right.set(port)
+                        }
```

Comment:
```suggestion
                        FpgaController::Left => physical_failure.left,
                        FpgaController::Right => physical_failure.right,
                    }
                    .set(port)
```
🤷🏻‍♂️

Reply:
On second thought, [do what I said above](https://github.com/oxidecomputer/hubris/pull/1239/files#r1154776203), then this becomes
```
physical_failure.get_mut(fpga_index).set(port);
```